tests/testthat/test-as_tibble.R

```diff
@@ -0,0 +1,178 @@
+t1 <- trial %>%
+  tbl_summary()
+
+t2 <- glm(response ~ age + grade, trial, family = binomial(link = "logit")) %>%
+  tbl_regression(exponentiate = TRUE)
+
+t3 <- tbl_uvregression(
+  trial %>% dplyr::select(response, age, grade),
+  method = glm,
+  y = response,
+  method.args = list(family = binomial),
+  exponentiate = TRUE
+)
+
+test_that("as_tibble works with standard use", {
+  expect_silent(res <- as_tibble(t1))
+  expect_silent(as_tibble(t1, return_calls = TRUE))
+  expect_snapshot(res %>% as.data.frame())
+
+  expect_silent(res <- as_tibble(t2))
+  expect_snapshot(res %>% as.data.frame())
+
+  expect_silent(res <- as_tibble(t3))
+  expect_snapshot(res %>% as.data.frame())
+})
+
+test_that("as_tibble(col_labels=) works", {
+  expect_equal(
+    as_tibble(t2, col_labels = FALSE) %>% names(),
+    c("label", "estimate", "conf.low", "p.value")
```

Comment:
@ddsjoberg is this label expected to be `"conf.low"` here (not `"ci"`)?

Reply:
That is correct. When the package was originally written gt didn't have column merging capabilities, and we'd manually create a new `ci` column. But we don't need to do that any more

Reply:
Sounds good! Thanks for clarifying